components/Calendar.js

```diff
@@ -76,11 +77,14 @@ export default class Calendar extends Component {
   componentDidUpdate() {
     this.scrollToItem(VIEW_INDEX);
   }
-  
+
   componentWillReceiveProps(props) {
     if (props.selectedDate) {
       this.setState({selectedMoment: props.selectedDate});
     }
+    if (props.currentMonth) {
+      this.setState({currentMonthMoment: moment(props.currentMonth)});
+    }
```

Comment:
Sorry if I wasnt totally clear with my intentions with this. My actual use case is to be able to set the currentMonth at any time. For example, in the same view as the calendar, I have a button that when pressed will reset the currently displayed month and date to show today's month/day. It would do this by changing the prop currentMonth of the calendar and that's what this PR enables.